### PR TITLE
add bulk mode to Model.load()

### PR DESCRIPTION
**Pull request by charbeljc**
add `bulk_mode=True` in your `context` before calling Model.load() and
your data is loaded within a single transaction instead of one per row.

### CODE REVIEW

File: openerp/models.py [hunk 1]
```diff
@@ -1088,8 +1099,15 @@ def load(self, cr, uid, fields, data, context=None):
...
+                if not bulk:
+                    cr.execute('ROLLBACK TO SAVEPOINT model_load_save')
+                else:
+                    break
+        if not bulk:
+            failures = ('error',)
+        else:
+            failures = ('error', 'warning')
+        if any(message['type'] in failures for message in messages):
             cr.execute('ROLLBACK TO SAVEPOINT model_load')
```
Comment (gurneyalex) [inline openerp/models.py:1111]: why is this one not protected with `if not bulk`?

**Comment by gurneyalex** [comment]
needs an automated test.

File: openerp/models.py (hunk 1 above)
Comment (charbeljc) [inline openerp/models.py:1111]: thanks for the catch

Comment (charbeljc) [inline openerp/models.py:1111]: in fact, this one does not depend of bulk.

**Comment by dreispt** [comment]
Should this be an addon module?

**Comment by pedrobaeza** [comment]
Closing as old and Travis status is red. I also consider this change very heavy for OCB. Any way, if there's still interest, please reopen and fix the thing.

**Pull request closed without merging**